spacksites/spack-env-templates/slurm/2025-09_youngmichael/avx2_buildtools.yaml

```
@@ -4,6 +4,13 @@ spack:
   view: false
 
   packages:
+    zlib:
+      require: 'zlib-ng'
```

If I understand correctly what you're trying to achieve here (using `zlib-ng` as zlib provider), the right way to do it is to choose `zlib-ng` as provider of the virtual package `zlib-api`, not `zlib`, which is a regular package (and an alternative provider of `zlib-api`) 
```suggestion
    zlib-api:
      require: 'zlib-ng'
```